src/gwkokab/models/utils/__init__.py

```
@@ -11,6 +11,7 @@
 from ._ncombination import (
     combine_distributions as combine_distributions,
     create_beta_distributions as create_beta_distributions,
+    create_independent_spin_orientation_gaussian_isotropic as create_independent_spin_orientation_gaussian_isotropic,
```

Pylint (3.3.7)

[convention] Line too long (117/100)

(C0301)